enterprise-metrics/scripts/k3d

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
I would not name this script `k3d` ;) since it may interfere with the real [`k3d`](https://k3d.io/)

Reply:
fair point, updated in https://github.com/grafana/jsonnet-libs/pull/598/commits/f4e41500cd5736eb21b5887cf4c57af5b97c464e